src/extension/features/feature.ts

```diff
@@ -48,7 +48,7 @@ export class Feature {
     /* stubbed listener function */
   }
 
-  onRouteChanged(): void {
+  onRouteChanged(route: any): void {
```

Comment:
do we need this type? i think if we're going to add this, it should be `string` not any.

in general, if we're adding types we should never use `any`, especially in our own code.

Reply:
I'll revert. Not sure why I added in the first place.

Reply:
Updated.